### ISSUE TITLE
Separate section for Direct messages

### ISSUE THREAD

**Issue by HerrHulaHoop** [sk22/megalodon#166]
**Is your feature request related to a problem? Please describe.**
The website view has a separate section where you can view all direct messages. While I know these messages aren't DM like other sites but regular toots restricted to be viewed only by mentioned users, it would be nice to have it be visible somewhere separately. Right now, the only clear indicator that a toot is a direct message is the Boost function being disabled.

**Describe the solution you'd like**
I would like if the app has a section to view all such direct messages. Where the exact placement should be, I'm not clear on. Will leave that to better minds 😄

**Comment by McKristoffer** [comment]
I agree with this wholeheartedly - maybe it could be a separate timeline tab (like how in the current version we have the "Community tab", "Federated tab", etc.)?

**Comment by neilideal** [comment]
I think it should be in notifications as a filter in the Mentions section

**Comment by sk22** [comment]
> I think it should be in notifications as a filter in the Mentions section. For example, in the "three dots" menu

filters definitely sound like a good idea, so i opened #294. though, i might still include a separate section/page for direct messages/conversations… i just don't yet know where and how

**Comment by sk22** [comment]
no need to spend a lot of time to draft something! (i'm a bit afraid that it might be a concept i don't want to or can't implement) maybe putting it into words could be enough if you want to share your idea? also, for reference, almost all icons are taken from microsoft's fluent icons

**Comment by neilideal** [comment]
> no need to spend a lot of time to draft something! (i'm a bit afraid that it might be a concept i don't want to or can't implement) maybe putting it into words could be enough if you want to share your idea? also, for reference, almost all icons are taken from microsoft's fluent icons

I don't remember how, but I was able to find mention of this set of icons and started looking. I'm much more interested in what kind of font is used and how the shadows on the different elements are set up.
About the sketch here on the one hand practice for me to improve my skills, and on the other the idea (direction vector) for you

**Comment by neilideal** [comment]
I made sketches on paper and sent them to you as a message visible only to you in mastodon

**Comment by McKristoffer** [comment]
Now that we have the drop down menu in the newest version with the Home, Local, Federated timelines, perhaps it could be placed there?

**Comment by sk22** [comment]
i think i'll actually move the "posts" section in the notification to the home tab (since "posts" is effectively a self-curated timeline) and add a notification tab named "direct"

**Comment by McKristoffer** [comment]
Yeah, with the new drop-down design it will probably be the most natural to put it here like so:
![image](https://user-images.githubusercontent.com/12572834/218312417-15189d79-41d7-4ada-ae15-102157a7e7ca.png)

**Comment by maop** [comment]
> Yeah, with the new drop-down design it will probably be the most natural to put it here like so:
> 
> ![image](https://user-images.githubusercontent.com/12572834/218312417-15189d79-41d7-4ada-ae15-102157a7e7ca.png)

Yes add this please!
Yesterday i spent hours trying to figure out how to see my "direct messages" and was surprised that Megalodon don't have this section. Then spent more hours on infinite scrolling till i found the conversation where the DM's where. This feature is very needed please :pray:
